src/Components/FooterNavigation.js

```diff
@@ -45,13 +45,14 @@ const NavBox = styled.div`
   z-index: 1000;
   .active {
     background: var(--secondary);
-    opacity: 0.9;
+    opacity: 0.8;
   }
 `;
 
 const NavLinkStyled = styled(NavLink)`
   background: transparent;
-  box-shadow: 0 0.125rem 0.75rem hsla(213, 52%, 20%, 0.2);
+  //box-shadow: 0 0.125rem 0.75rem hsla(213, 52%, 20%, 0.2);
```

Comment:
you can remove this comment (:

Reply:
Thank you for the hint! Consider it done. :)